api-docs/classic-api-migration-guide.md

```diff
@@ -0,0 +1,81 @@
+
+The [Classic Group API](https://dev.trackunit.com/docs/group) allowed you to create and manage collections of units. Because Iris is build around the concept of assets, the [Iris Group API](https://developers.trackunit.com/reference/getgroups) allows you now to create and manage collections of assets instead. Additionally the Iris Group API endpoints also allow you to manage which users should have access to groups.
+
+*(any migration tool here for groups?!)*
+*+ what about Classic Category?!*
+
+
+### Sites API
+With the Iris platform we have evolved the concept of [Classic Zones](https://dev.trackunit.com/docs/zone) (map areas) and [Classic Points](https://dev.trackunit.com/docs/point) (location points on a map) and collapsed the two into our new Sites concept. Sites keep the same importance as zones for running reports and creating alerts. Leverage automatically detected sites or enter site boundaries as you see fit – including the option to apply labels depending on the site type.
+
+With the introduction of Sites in Trackunit Manager, existing Zones in Manager Classic were automatically transferred to Sites and classified as Classic Zones. To harvest the full power of Sites, it is recommended to [manually convert a Zone into a Site type](https://help.trackunit.com/en/articles/138565-how-do-i-convert-a-zone-manager-classic-to-a-site-in-trackunit-manager) of your choice and get an up-to-the-minute picture of equipment status across job sites.
+
+Going forward use the [Iris Sites API](https://developers.trackunit.com/reference/getsites) to create and manage your Sites at scale and also e.g. retrieve a comprehensive site history, encompassing either asset intervals within a list of sites or the sites visited by a list of assets.
+
+## Organizational Management APIs
+
+### Customers API
+The [Classic Clients](https://dev.trackunit.com/docs/client) & [Contacts](https://dev.trackunit.com/docs/contact) functionality now has an equivalent feature called Customers in Trackunit Iris. We recommend you start using Customers as soon as possible to get access to the latest functionalities and be on board with the foundation for exiting new features like [ERP integrations](https://portal.productboard.com/mbaayvr5tzubn5acbd8dvqa8/c/288-rental-erp-integrations) and [Branded Customer Portals](https://portal.productboard.com/mbaayvr5tzubn5acbd8dvqa8/c/290-branded-customer-portals).
+
+> 🚧 Migration note
+> If you have set up Clients in Manager Classic that are attached to Service Management and Alarms in Manager Classic, we recommend that you wait until those two features are completely migrated over to Trackunit Iris to avoid disrupting your business processes. If you are in doubt, contact your Trackunit representative.
+
+*(any migration tool for clients?)*
```

Comment:
The migration tool can now also migrate clients. This is behind feature flag. I'm not sure what the launch plan is.